packages/terra-menu/src/Menu.jsx

```diff
@@ -120,6 +216,8 @@ class Menu extends React.Component {
       targetRef,
       isArrowDisplayed,
       contentWidth,
+      rootSelector,
```

Comment:
I'd not extract these if they ain't used within `render`.

Reply:
Fixed in  bad1e3c